gap/tools/io.gi

```diff
@@ -126,7 +126,7 @@ function(arg)
   if Length(arg) = 2 then
     name := arg[1];
     coll := arg[2];
-    mode := "w";
+    mode := "w"; #FIXME: Doc says this should be "a" by default.
```

Comment:
Should I change the doc for `WriteGenerators` to indicate that the default option is "w" ?